server/testutils/factories/risks.ts

```diff
@@ -10,13 +10,13 @@ const riskEnvelopeStatuses: Array<RiskEnvelopeStatus> = ['retrieved', 'not_found
 
 export default Factory.define<PersonRisks>(() => ({
   crn: `C${faker.number.int({ min: 100000, max: 999999 })}`,
-  roshRisks: roshRisksFactory.build(),
+  roshRisks: roshRisksEnvelopeFactory.build(),
   mappa: mappaFactory.build(),
   flags: flagsFactory.build(),
   tier: tierEnvelopeFactory.build(),
 }))
 
-const roshRisksFactory = Factory.define<PersonRisks['roshRisks']>(() => ({
+export const roshRisksEnvelopeFactory = Factory.define<PersonRisks['roshRisks']>(() => ({
```

Comment:
maybe the commit message here ought to be tweaked now that your have introduced a "risks envelope factory". ie. rather than

> The risk factories file here contains much more complexity than we need for our service, but because we are calling an API endpoint that returns types of risk data that we don’t need, for now we need to retain this complexity.

maybe something to the effect that you've opted to return a subset of the data from the `PersonService.getRoshRisks()` and introduce a more specialised factory?

Reply:
yes, although I've actually just renamed the existing `roshRisksFactory` to be more specific. The complexity comment is more referring to the file in general, which contains factories for various bits of data that are required by the application type but not by our service. But I could just remove that comment if it's confusing?